releng/metaborg/releng/build.py

```diff
@@ -393,6 +394,21 @@ def __build_intellij(basedir, gradle, **_):
       ),
     ])
 
+  @staticmethod
+  def __build_spt_intellij(basedir, gradle, **_):
+    target = 'install'
+    cwd = os.path.join(basedir, 'spt', 'org.metaborg.spt.testrunner.intellij')
+    gradle.run_in_dir(cwd, target)
+    return StepResult([
+      MetaborgFileArtifact(
+        'SPT test runner for IntelliJ',
+        _glob_one(os.path.join(basedir, cwd, 'build', 'distributions', 'org.metaborg.spt.testrunner.intellij-*.zip')),
+        os.path.join('spt', 'intellij', 'plugin.zip'),
+        NexusMetadata('org.metaborg', 'org.metaborg.spt.testrunner.intellij'),
+        BintrayMetadata('spt-intellij-updatesite')
```

Comment:
I'm not sure about the target path, Nexus and Bintray metadata. Is this acceptable?